templates/docs/examples/layouts/full-width/default.html

```diff
@@ -0,0 +1,224 @@
+        <li class="p-navigation__item">
+          <button class="js-search-button p-navigation__link--search-toggle">
+            <span class="p-navigation__search-label">Search</span>
+          </button>
+        </li>
+        <li class="p-navigation__item">
+          <button class="js-menu-button p-navigation__link">Menu</button>
+        </li>
+      </ul>
+    </div>
+    </div>
+    <div class="l-main">
+      <div class="p-navigation__row u-fixed-width">
+    <nav class="p-navigation__nav" aria-label="Main">
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item is-selected"><a class="p-navigation__link" href="/docs/examples/layouts/full-width/default">Docs</a></li>
+        <li class="p-navigation__item "><a class="p-navigation__link" href="/docs/examples/layouts/full-width/default">Examples</a></li>
+        <li class="p-navigation__item"><a class="p-navigation__link" href="/docs/examples/layouts/full-width/default">Accessibility</a></li>
+        <li class="p-navigation__item"><a class="p-navigation__link" href="/docs/examples/layouts/full-width/default">Browser support</a></li>
+        <li class="p-navigation__item"><a class="p-navigation__link" href="/docs/examples/layouts/full-width/default">Contribute</a></li>
+      </ul>
+      <ul class="p-navigation__items">
+        <li class="p-navigation__item">
+          <button class="js-search-button p-navigation__link--search-toggle">
+            <span class="p-navigation__search-label">Search</span>
+          </button>
+        </li>
+      </ul>
+      <div class="p-navigation__search">
+        <form class="p-search-box" action="/docs/search">
+          <input type="search" id="search-docs" class="p-search-box__input" name="q"  placeholder="Search the docs" title="Search the documentation"  autocomplete="on" aria-label="Search the docs" required />
+          <button type="reset" id="search-docs-reset" class="p-search-box__reset u-no-margin--right" name="close"><i class="p-icon--close">Clear input</i></button>
+          <button type="submit" class="p-search-box__button" name="submit"><i class="p-icon--search">Submit</i></button>
+        </form>
+      </div>
+    </nav>
+  </div>
+  </div>
+    <div class="l-end">
+    <nav class="p-navigation__nav p-navigation--right" style="width: auto">
+      <ul class="p-navigation__items" style="margin-left: auto">
+        <li class="p-navigation__item"><a class="p-navigation__link" href="/docs">Login</a></li>
+      </ul>
+    </nav>
+    </div>
+  </div>
+  <div class="p-navigation__search-overlay"></div>
+</header>
+
+<div class="l-full-width__sidebar" style="background: #f7f7f7">
+
+  <div class="p-side-navigation is-sticky" id="side-navigation-drawer" aria-label="Side" style="background: #f7f7f7">
+
+    <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="side-navigation-drawer"></div>
+
+    <nav class="p-side-navigation__drawer" aria-label="Example side"  style="background: #f7f7f7">
```

Comment:
```suggestion
<div class="l-full-width__sidebar">

  <div class="p-side-navigation is-sticky" id="side-navigation-drawer" aria-label="Side">

    <div class="p-side-navigation__overlay js-drawer-toggle" aria-controls="side-navigation-drawer"></div>

    <nav class="p-side-navigation__drawer" aria-label="Example side"  style="background: #f7f7f7">
```
The background colour is set multiple time. It's redundant, I don't know if you want to keep it